### ISSUE TITLE
[Accessibility] ‘Road’ drop down is not getting collapsed by pressing esc key.

### ISSUE THREAD

**Issue by cdhawan** [Microsoft/BingMapsV8CodeSamples#35]
Keyboard user will not be able to collapse the control and cannot access the control efficiently if the control is not collapsing through by pressing Esc key.

**Comment by drwestco** [comment]
This repo is for the V8 Code Samples code snippets, not the Bing Map SDK control itself. Please use the feedback mechanism within Bing Maps to report this issue.
